src/Console/Command/DescribeCommand.php

```diff
@@ -160,7 +161,7 @@ private function describeRule(OutputInterface $output, $name)
             $output->writeln(sprintf('Fixer is configurable using following option%s:', 1 === count($options) ? '' : 's'));
 
             foreach ($options as $option) {
-                $line = '* <info>'.$option->getName().'</info>';
+                $line = '* <info>'.OutputFormatter::escape($option->getName()).'</info>';
```

Comment:
is it needed?

Reply:
Yes, I believe so. Even if SF gets patched we would've to black list the bugged versions otherwise. Beside the fixes is most likely going to be changing the escape sequence from `<<<` to some non printable char, however if we use that char we would end up with the same problem.
Escaping here will resolve all those issues.

Reply:
why non-printable char would be an issue for us...
ok, let it stay like it is, but I still extra confused about it...
maybe we could at least add some test that would fail without escaping on broken symfony/console version ?

Reply:
I don't understand, the tests did fail with SF console 3.0.4, that is why I changed it so the tests are green again

Reply:
oh, OK. I was not able to reproduce test failure localy

Reply:
about "\0" issue might, be resolved here; https://github.com/symfony/symfony/pull/23730#discussion_r135804540

anyway, escaping the output still seems like a good thing to do to prevent other regressions and issues (and we don't have to exclude known SF console releases with the issue)

Reply:
To reproduce locally:

```
[12:55 PM]-[possum@zoo-playground-1]-[~/lala] 
$ cat test.php 
<?php
require_once __DIR__.'/vendor/autoload.php';
$output = new Symfony\Component\Console\Output\ConsoleOutput();
$output->writeln('[<<]');
die;
[12:55 PM]-[possum@zoo-playground-1]-[~/lala] 
$ composer require symfony/console 3.0.0
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Package operations: 0 installs, 1 update, 0 removals
  - Updating symfony/console (v3.0.4 => v3.0.0): Loading from cache
Writing lock file
Generating autoload files

[12:55 PM]-[possum@zoo-playground-1]-[~/lala] 
$ php test.php 
[<<]

[12:55 PM]-[possum@zoo-playground-1]-[~/lala] 
$ composer require symfony/console 3.0.4
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Package operations: 0 installs, 1 update, 0 removals
  - Updating symfony/console (v3.0.0 => v3.0.4): Loading from cache
Writing lock file
Generating autoload files

[12:56 PM]-[possum@zoo-playground-1]-[~/lala] 
$ composer info -D|grep symfony/console
1:symfony/console v3.0.4 Symfony Console Component

[12:56 PM]-[possum@zoo-playground-1]-[~/lala] 
$ php test.php 
[\]
```

Reply:
btw, symfony/symfony#23730 just got merged